contrib/scalar/scalar.c

```diff
@@ -68,6 +76,24 @@ static void setup_enlistment_directory(int argc, const char **argv,
 	}
 
 	setup_git_directory();
+
+	/* Scalar requires the use of a credential helper */
+	setenv("GIT_TERMINAL_PROMPT", "false", 0);
+	if (is_unattended())
+		git_config_push_parameter("credential.interactive=never");
```

Comment:
Just as an FYI `credential.interactive` is specific to GCM4W and GCMC.. other helpers do not care for this setting.
Therefore, there is an implicit assumption the configured helper for Scalar is GCM. Might want to call this out, or make it a piece of work to ensure this assumption is removed.

Reply:
Oh, right...

Hmm. I really do not know what else we could do to prevent that authenticated `vsts/info` endpoint from _at least_ producing ugly "error: config: (http:401) Not Authorized" lines.

I guess we could special-case Scalar's functional tests, by setting an environment variable that will skip looking at that endpoint. Probably what we should do. Do you concur?